### ISSUE TITLE
Modrinth Support

### ISSUE THREAD

**Issue by Ecorous** [shedaniel/RoughlyEnoughItems#694]

**Comment by shedaniel** [comment]
Read my stance on modrinth at https://github.com/shedaniel/cloth-api/issues/38

**Comment by Ecorous** [comment]
I find this really really stupid. Why do modders only care about the money?

**Comment by ThexXTURBOXx** [comment]
They don't "only" care about the money.
For example, I never cared about money since I neither develop Minecraft mods for a living nor are my mods that popular. Also, I work at a university and don't earn too little money.
However, I completely understand everyone who wants or needs this (maybe even extra) money.
It's not reprehensible to collect money for develop mods for everyone which they can then use **for free**.

**Comment by shedaniel** [comment]
I don't only care about money, but money is a major factor on how long I can spend on modding. I average 20 hours on modding every week and it is unreasonable for me to continue to do so with my current financial situation if I cannot get a reasonable amount of money from this.

TL;DR I would rather allocate my time spent on modding on being a cashier if modding doesn't give me money

**Comment by Ecorous** [comment]
> I don't only care about money, but money is a major factor on how long I can spend on modding. I average 20 hours on modding every week and it is unreasonable for me to continue to do so with my current financial situation if I cannot get a reasonable amount of money from this.
> 
> TL;DR I would rather allocate my time spent on modding on being a cashier if modding doesn't give me money

Sorry if I'm just stupid, but what I just read (to me) was "I don't care about the money but I care about the money"

**Comment by ThexXTURBOXx** [comment]
You maybe missed one crucial word:

I don't **only** care about money

**Comment by Ecorous** [comment]
Oops! That was me being stupid! Sorry y'all!
